src/librustc/lint/mod.rs

```diff
@@ -74,25 +75,46 @@ pub struct Lint {
     ///
     /// e.g. "imports that are never used"
     pub desc: &'static str,
+
+    /// Deny lint after this epoch
+    pub epoch_deny: Option<config::Epoch>,
```

Comment:
Further down you state that the lint should be removed after the epoch, here it seems to suggest it should be `deny`.

Reply:
Where is this? It used to say this, but that changed.

This specific lint becomes Deny in Rust2018. Other lints may wish to disappear entirely because they are no longer lints.